### PR TITLE
Structureharvester

### PR DESCRIPTION
**Pull request by lecorguille**
Hop,
A new tool which can come after [structure](https://github.com/galaxyproject/tools-iuc/tree/master/tools/structure)

### CODE REVIEW

File: tools/structureharvester/test-data/summary.html [hunk 1]
```diff
@@ -0,0 +1,146 @@
+
```
Comment (bgruening) [inline tools/structureharvester/test-data/summary.html]: This file is not used in the test, isn't it?

File: tools/structureharvester/test-data/example_harvester_archive/K1ReRun_run_2_f [hunk 2]
```diff
@@ -0,0 +1,384 @@
+
+
```
Comment (bgruening) [inline tools/structureharvester/test-data/example_harvester_archive/K1ReRun_run_2_f]: Can we use the archive from the structure tool as symlink?

File: tools/structureharvester/test-data/summary.html (hunk 1 above)
Comment (lecorguille) [reply tools/structureharvester/test-data/summary.html]: Indeed, I took it from the developer website and expect to obtain something similar but no, only a boring text file. I will remove it

File: tools/structureharvester/test-data/example_harvester_archive/K1ReRun_run_2_f (hunk 2 above)
Comment (lecorguille) [reply tools/structureharvester/test-data/example_harvester_archive/K1ReRun_run_2_f]: I need more outputs than the one I generate for the structure test.
I can reduce a little more the inputs in term of number if you want?

Comment (bgruening) [reply tools/structureharvester/test-data/example_harvester_archive/K1ReRun_run_2_f]: Its ok I think was just wondering.

File: tools/structureharvester/test-data/summary.html (hunk 1 above)
Comment (bgruening) [reply tools/structureharvester/test-data/summary.html]: Thanks!

**Comment by lecorguille** [comment]
Done! 
Thanks for your review @bgruening

**Comment by bgruening** [comment]
@lecorguille can I squash and merge or do you want to quash before merging?

**Comment by lecorguille** [comment]
Done
(Damn! My rank 😄 )

**Comment by bgruening** [comment]
@lecorguille you can only squash the latest commit to get rid of this one file, not everything. Sorry should have told you earlier.

**Comment by lecorguille** [comment]
^^'

**Pull request merged**